config/webpacker.yml

```diff
@@ -10,7 +10,8 @@ default: &default
 
   # Additional paths webpack should look up modules
   # ['app/assets', 'engine/foo/app/assets']
-  additional_paths: []
+  additional_paths:
+    - public
```

Comment:
For some reason needed for `background: url("images/blacklisted-preview.png");` in scss.